4_operations.html

```diff
@@ -0,0 +1,31 @@
+<meta charset="utf-8">
+<script type="text/javascript">
+	
+	var a = +prompt("Введите число a")
+	var b = +prompt("Введите число b")
+
+	function add (a,b) {
+		return a+b
```

Comment:
Между операторами и операндами лучше ставить пробелы для лучшей читаемости кода
```suggestion
		return a + b
```

Reply:
Проставьте, пожалуйста, их везде